src/chrome/content/rules/Senate.gov.xml

```diff
@@ -15,7 +15,10 @@
 
 		<!-- The rules subdomain is all 403 via TLS -->
 		<!-- As is the Senate Intelligence Committee -->
-		<exclusion pattern="^http://(?:www.)?(?:intelligence|rules)\.senate\.gov/" />
+		<exclusion pattern="^http://(?:www.)?intelligence\.senate\.gov/" />
+		<exclusion pattern="^http://(?:www.)?rules\.senate\.gov/" />
```

Comment:
It is better to specify explicit routes rather than conditional regex, as
the style guide specifies.

The other change is necessary to view Rep. Sessions' page.
On Mar 3, 2016 3:11 PM, "loveithateit" <notifications@github.com> wrote:

> In src/chrome/content/rules/Senate.gov.xml
> <https://github.com/EFForg/https-everywhere/pull/4260#discussion_r54964938>
> :
>
> > @@ -15,7 +15,10 @@
> >
> >  		<!-- The rules subdomain is all 403 via TLS -->
> >  		<!-- As is the Senate Intelligence Committee -->
> > -		<exclusion pattern="^http://(?:www.)?(?:intelligence|rules)\.senate\.gov/" />
> > +		<exclusion pattern="^http://(?:www.)?intelligence\.senate\.gov/" />
> > +		<exclusion pattern="^http://(?:www.)?rules\.senate\.gov/" />
>
> Is this change really necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EFForg/https-everywhere/pull/4260/files#r54964938>.
>